src/pages/metapackages/venia-sample-data/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Venia Sample Data metapackage
+---
+# Venia Sample Data metapackage
+
+This metapackage contains Venia Sample Data modules you can install to help you develop your projects locally using sample data.
+
```

Comment:
Add a statement saying Open Source must be installed already. Even if that statement isn't technically true, it's real world.